### PR TITLE
FB-969 Don't rely on DNS check in unit tests

### PR DESCRIPTION
**Pull request by dlondero**

### CODE REVIEW

File: library/CM/Service/NetworkTools.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+class CM_Service_NetworkTools {
+
+    /**
+     * @param string $domain
+     * @return bool
+     */
+    public function getMXRecords($domain) {
```
Comment (fauvel) [inline library/CM/Service/NetworkTools.php]: It is rather `hasMXRecords` if you only return a boolean and dismiss the host list...

**Comment by fauvel** [comment]
Otherwise lgtm!

**Comment by dlondero** [comment]
Fixed naming, updated tests.

**Comment by fauvel** [comment]
Lgtm!

**Pull request merged**